target/linux/generic/config-6.6

```diff
@@ -834,6 +834,8 @@ CONFIG_CARDBUS=y
 # CONFIG_CC10001_ADC is not set
 # CONFIG_CCS811 is not set
 CONFIG_CC_CAN_LINK=y
+CONFIG_CC_IMPLICIT_FALLTHROUGH="-Wimplicit-fallthrough=5"
+CONFIG_CC_NO_ARRAY_BOUNDS=y
```

Comment:
All these `CONFIG_(G)CC_*` values (apart from `CC_OPTIMIZE_FOR_*`) are dynamically set based on the used compiler (version), and should be moved to https://github.com/openwrt/openwrt/blob/main/target/linux/generic/config-filter so they don't show up even if their values may change due to switching compiler versions.

Reply:
Done. Fixed up my other commit too.